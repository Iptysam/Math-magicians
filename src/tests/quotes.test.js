import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../component/Pages/qoute';

it('It should properly render the quotes page', () => {
  const tree = renderer
    .create(<Quote />).toJSON();

  expect(tree).toMatchSnapshot();
});
