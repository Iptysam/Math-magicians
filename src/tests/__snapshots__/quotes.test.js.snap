// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check on quotes page It should properly render the quotes page 1`] = `
<p
  className="qoutes"
>
  Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality.
</p>
`;

exports[`It should properly render the quotes page 1`] = `
<p
  className="qoutes"
>
  Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality.
</p>
`;

exports[`Test for the  quotes page It should properly render the quotes page 1`] = `
<p
  className="qoutes"
>
  Mathematics as an expression of the human mind reflects the active will, the contemplative reason, and the desire for aesthetic perfection. Its basic elements are logic and intuition, analysis and construction, generality and individuality.
</p>
`;

exports[`Test for the quotes page navigation`] = `<p className="qoutes">
    Mathematics as an expression of the human mind reflects the active will,
    the contemplative reason, and the desire for aesthetic perfection.
    Its basic elements are logic and intuition, analysis and construction,
    generality and individuality.
  </p>`;
